### DISCUSSION TITLE
Forms API - Link Documents in ProjectWise and other forms Forms

### DISCUSSION THREAD

**Discussion by inigolorente** [iTwin/itwinjs-core#5659] [Q&A]
Hello,
I am trying to use the iTwin Forms API to create a Form linked to a couple documents in ProjectWise, linked to other forms as well.
I have read the iTwin documentation: https://developer.bentley.com/apis/forms/operations/create-form-data/
But there is no request example to create a form like the following, with 2 links to PW documents and 1 related form like the one shown below/attached.

Do you have some example payload for creating a form like that?

![FormLinks](https://github.com/iTwin/itwinjs-core/assets/81552659/c9502da0-c8c6-4e6b-b76c-f4c2b84fd23a)

Thanks in advance for your support :)

**Comment by pratikshan85** [comment]
@inigolorente We are looking into the issue.

**Reply by pratikshan85** [reply]
@inigolorente Currently we don't have any API which can resolve your requirement. 
From UI you can link files to your form.

**Comment by inigolorente** [comment]
Oh... it would be very useful for us to have a feature like this. I have filled an "idea" in the forum: https://github.com/iTwin/itwinjs-core/discussions/5670

Thanks anyways
________________________________
De: pratikshan85 ***@***.***>
Enviado: Thursday, June 22, 2023 8:09:53 AM
Para: iTwin/itwinjs-core ***@***.***>
Cc: Lorente Riverola, Iñigo (SMO SWE ST BIM) ***@***.***>; Mention ***@***.***>
Asunto: Re: [iTwin/itwinjs-core] Forms API - Link Documents in ProjectWise and other forms Forms (Discussion #5659)


@inigolorente<https://github.com/inigolorente> Currently we don't have any API which can resolve your requirement.
From UI you can link files to your form.
